### PR TITLE
[NABI-65] menu drawer and  buttons

### PR DESCRIPTION
**Pull request by vickcharles**
Implements [NABI-65](https://iamvane.backlog.com/view/NABI-65)

**Main task:**

- Set up drawer in the header
- Add drawer button to header
- When user clicks on the button a drawer displays

### CODE REVIEW

**Comment by vickcharles** [comment]
@iamvane hey vane, i'm waiting for feedbacks

File: src/assets/css/index.css [hunk 1]
```diff
@@ -182,194 +190,315 @@ li {
...
+.nabi-header-button {
+	position: absolute !important;
+	top: 8% !important;
+	right: 0% !important;
+}
+
+.nabi-header-button .create-account-button {
+	padding: 3px 14px !important;
+	font-size: 10px;
+	border-radius: 0px !important;
```
Comment (iamvane) [inline src/assets/css/index.css]: Remove border-radius. The existing border style on the material-ui buttons is what we want to use.

File: src/assets/css/index.css [hunk 2]
```diff
@@ -182,194 +190,315 @@ li {
...
+}
+
+.nabi-header-button .login-button {
+	padding: 3px 14px !important;
+	font-size: 10px;
+	border-radius: 0px !important;
```
Comment (iamvane) [inline src/assets/css/index.css]: Same as before. Copy/paste: Remove border-radius. The existing border style on the material-ui buttons is what we want to use.

File: src/ts/main/components/Header.tsx [hunk 3]
```diff
@@ -1,28 +1,40 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
-
-/** 
- * Header component
- */
+import DrawerMenu from './drawer';
+import Button from 'material-ui/Button';
```
Comment (iamvane) [inline src/ts/main/components/Header.tsx]: Add linespace after line 4

File: src/assets/css/index.css [hunk 4]
```diff
@@ -182,194 +190,315 @@ li {
...
+	position: relative !important;
+	width: 90%;
+	height: auto;
+	margin-left: auto;
+	margin-right: auto;
+	padding-bottom: 10px;
+	padding-top: 65px;
+}
+
+.nabi-header-line {
```
Comment (iamvane) [inline src/assets/css/index.css]: Here we should use the Divider component from material-ui and add the class name "hide-on-desktop" to hide it on bigger screens. Please delete this css rule and its implementation.

File: src/assets/css/index.css [hunk 5]
```diff
@@ -182,194 +190,315 @@ li {
...
-    width: 990px;
-  }
-}
+
+@media only screen and (min-width: 1200px) {
+	/***** HOME *****/
+	.nabi-container {
+		width: 990px;
+	}
+}
```
Comment (iamvane) [inline src/assets/css/index.css]: fix no newline error

**Pull request merged**